src/Core/Domain/Attachment/QueryResult/EditableAttachment.php

```diff
@@ -44,7 +44,7 @@ class EditableAttachment
     private $name;
 
     /**
-     * @var string[]|null
+     * @var string[]
```

Comment:
Same here, same logic: Edit Command have nullable fields and it's done on purpose 😉

Reply:
A null parameter will trigger a fatal error : 
![image](https://user-images.githubusercontent.com/1533248/89201294-8bd2d800-d5b1-11ea-9c17-fd7d75d5f332.png)